ministryofjustice/analytics-platform-control-panel
# App bedrock access

## :memo: Summary
This PR resolves ministryofjustice/analytical-platform#4311

This PR allows bedrock policy to be applied to an app role

## :mag: What should the reviewer concentrate on?
- Feedback on specific parts of the code
- Check side effects, if any

## :technologist: How should the reviewer test these changes?
- Pull down branch and run locally
- Go to app management page and check that bedrock checkbox exists
- Toggle checkbox and check that bedrock policy is applied/removed appropriately

## :books: Documentation status
